src/log.js

```diff
@@ -1,8 +1,12 @@
+const { _postFormatInterceptors } = require('./config')
 const { logLevels, getLogLevel } = require('./levels')
 const { format } = require('./format')
 
 const doLog = (level, ...args) => {
-  const output = format(level, ...args)
+  let output = format(level, ...args)
+  _postFormatInterceptors.forEach(interceptor => {
```

Comment:
Perhaps add a test case to one (or each) of the log methods just to be absolutely sure this works as expected.